src/supermarket/config/locales/en.yml

```diff
@@ -96,6 +96,10 @@ en:
       unsubscribe:
         failure: "There was an issue unsubscribing Supermarket from the repo."
         success: "Supermarket is no longer monitoring this repo."
+  quality_metrics:
+    collaborator:
+      success: 'passed with %{collaborators}.'
+      failure: 'Failure: Cookbook has %{collaborators}. A cookbook must have at least %{passing_number} collaborators to pass this metric.'
```

Comment:
I think these entries ought to go into `src/fieri/config/locales/en.yml` for the engine to own the messages.

Also, I suggest the interpolated variable here be `num_collaborators` for both messages and move the string "collaborators" to display after the number from the CollaboratorWorker calls to I18n to here.